### ISSUE TITLE
Improvement suggestion - not just for ETW

### ISSUE THREAD

**Issue by pieter-venter** [dotnet/dotnet-api-docs#7311]
https://github.com/dotnet/dotnet-api-docs/blob/0e12f0a0ebfc81babdec8f02793ac9dbc4a7ab11/xml/ns-System.Diagnostics.Tracing.xml#L3

The description for the Namespace is quite misleading. It would seem like the classes here are just for ETW, but in actual fact, EventSource, EventCounter etc in this namespace are specifically meant to be cross platform.

**Comment by steveharter** [comment]
cc @ViktorHofer @tarekgh @Anipik

**Comment by tarekgh** [comment]
CC @noahfalk @josalem

**Comment by noahfalk** [comment]
Agreed, we should change the text. EventSource works with ETW, EventListener, EventPipe, and LTTng at this point.
